### ISSUE TITLE
Automatically deduce region. 

### ISSUE THREAD

**Issue by charlesbalderson** [JosephRedfern/TheAuditTrail#20]
Rather than hardcode a sections region, we should add a polygon field to the `Regions` model. 

We would be able to determine which regions a section intersects. It would make it faster and more accurate when creating sections.

Although quite an expensive query we could cache it on the model. 

`section.regions`

It would also allow us to show some funky stats on a path/section/region without too much effort. 
We also need to establish what a region is? I was thinking counties and national parks?

Probably wouldn't be too much effort to get hold of kml of the boundaries.

**Comment by JosephRedfern** [comment]
Worth noting that some section are in two regions at once, particularly around the south anglesey coast and the mainland.

**Comment by charlesbalderson** [comment]
I guess that wouldn't make too much of a difference? Just see which region a section intersects?

**Comment by JosephRedfern** [comment]
Yeah, this is true. I'm not sure exactly how accurate the official maps are - but I'm sure there's a way around it.
